nbs/06_transform.ipynb

```diff
@@ -21,7 +21,16 @@
    "cell_type": "code",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.                    "Input node 2 was removed by pattern node z, which conflicts with preserved pattern node y",
</pre><p>It wasn't removed. We noticed a conflict and so we didn't remove it.</p><p>Change error messages accordingly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>